kroxylicious-filters/kroxylicious-record-encryption/src/test/java/io/kroxylicious/filter/encryption/RecordEncryptionFilterTest.java

```diff
@@ -82,7 +87,7 @@
 @ExtendWith(MockitoExtension.class)
 class RecordEncryptionFilterTest {
 
-    private static final String UNENCRYPTED_TOPIC = "unencrypted";
+    private static final String UNRESOLVED_TOPIC = "unresolved";
```

Comment:
Deleting `UNENCRYPTED_TOPIC`  implies to me that we are missing a test case where the topic exists but doesn't have encryption applied.